android/app/src/main/java/ru/spbstu/amd/learnbraille/screens/practice/PracticeFragment.kt

```diff
@@ -82,12 +105,31 @@ class PracticeFragment : Fragment() {
             viewModel.onCorrectComplete()
         })
 
+        hintButton.setOnClickListener {
+            viewModel.onHint()
+            Toast.makeText(context, viewModel.getDotsString(), Toast.LENGTH_SHORT).show()
+            Timber.i("Hint invoked")
+            val expectedDots = viewModel.getExpectedDots()
+            Timber.i(expectedDots.toString())

```

Comment:
This log message is useless without commentary.

Reply:
Gather it with previous log message

Reply:
got it